### PR TITLE
Feat: API gateway for Question Service

### PR DESCRIPTION
**Pull request by HaroldCuico**
Added an API gateway for the question service

### CODE REVIEW

File: WebApp/backend/QuizMaster.API.Gatewway/Controllers/QuestionGatewayController.cs
Comment (jaymar921) [inline WebApp/backend/QuizMaster.API.Gatewway/Controllers/QuestionGatewayController.cs:1]: all good

**Review by JMark-FS — APPROVED** [review]
Looks good

File: WebApp/backend/QuizMaster.API.Quiz/Services/GRPC/QuestionService.cs
Comment (jaymar921) [inline WebApp/backend/QuizMaster.API.Quiz/Services/GRPC/QuestionService.cs:1]: good

**Review by jaymar921 — APPROVED** [review]
All goods

**Pull request merged**